funnel/templates/forms.html.jinja2

```diff
@@ -8,7 +8,7 @@
     {%- endif %}
 {%- endmacro %}
 
-{% macro renderfield(field, css_class="", widget_css_class="", sidetext=none, tabindex=none, autofocus=false, nolabel=false, style='horiz', rows=none) -%}
+{% macro renderfield(field, css_class="", widget_css_class="", sidetext=none, tabindex=none, autofocus=false, nolabel=false, style='horiz', rows=none, placeholder="") -%}
```

Comment:
This should be `placeholder=none` so that if no placeholder is specified, the attribute will be skipped. Default `=''` will render as `placeholder=''` on all widgets.